src/App.jsx

```diff
@@ -14,33 +16,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [sortField, setSortField] = useState('');
+  const [reverse, setReverse] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const sortGoods = sortBy => {
```

Comment:
Agreed with Nazar. Please feel free to ask for help in fe_chat.
`You can combine all of these functions into one and pass goods, sortType, and reverse state to it and based on them you will calculate a new goods array`